### DISCUSSION TITLE
Why does individual run from UI only work with Celery/K8Executor? Why not DaskExecutor?

### DISCUSSION THREAD

**Discussion by lformant** [apache/airflow#17762] [Q&A] [answered]
What precludes the DaskExecutor from being valid for run submission from the UI?

https://github.com/apache/airflow/blob/c4d043d9f8b28f2ad80c9912e7690bc6620bd753/airflow/www/views.py#L1485

Thanks.

**Comment by ashb** [comment, marked as answer]
Likely nothing, this check likely predates the addition of the DaskExecutor and no one has gone to change it.

**Reply by lformant** [reply]
Cool, any chance DaskExecutor could be greenlit in the next version?

**Reply by potiuk** [reply]
You can make your PR to fix it @lformant . This has the highest chance of success if you lead it to completion. Airflow is a community-driven  project so anyone can contribute.

**Reply by ghost** [reply]
Can confirm this works just fine by adding a section mimicking the Celery/KubernetesExecutor import try/excepts.

**Reply by lformant** [reply]
Great, @aa1371 since you have a working solution would you be willing to open the PR?

**Reply by ghost** [reply]
Sure I don't mind, but if you want to create the PR yourself go for it. The change is pretty straightforward. You can just copy the try/except from CeleryExecutor and change occurrences of celery -> dask
